README.md

```diff
@@ -148,207 +174,247 @@ Klaviyo.createEvent(event)
 
 ## Push Notifications
 
-### Prerequisites:
+### Prerequisites
+- A [Firebase account](https://firebase.google.com/docs/android/setup) for your Android app
+- Familiarity with [Firebase](https://firebase.google.com/docs/cloud-messaging/android/client) documentation
+- Configure [Android push](https://help.klaviyo.com/hc/en-us/articles/14750928993307) in your Klaviyo account settings
 
-- Firebase account
-- Familiarity with [Firebase](https://firebase.google.com/docs/cloud-messaging/android/client)
-  documentation.
-
-### KlaviyoPushService
-
-The Klaviyo Push SDK for Android works as a wrapper around `FirebaseMessagingService` so the
-setup process is very similar to the Firebase client documentation linked above.
-You should follow all other setup recommendations from the FCM documentation.
-Register `KlaviyoPushService` to receive `MESSAGING_EVENT` intents. This allows Klaviyo's Push SDK
-to receive new and updated push tokens via the `onNewToken` method,
-as well as display notifications via the `onMessageReceived` method.
+### Setup
+The Klaviyo Push SDK for Android works as a wrapper around `FirebaseMessagingService`, so the
+setup process is very similar to the Firebase client documentation linked above.  
+In your `AndroidManifest.xml` file, register `KlaviyoPushService` to receive `MESSAGING_EVENT` intents.
 
 ```xml
+<!-- AndroidManifest.xml -->
 <service android:name="com.klaviyo.pushFcm.KlaviyoPushService" android:exported="false">
```

Comment:
minor improvement here is that this is nested within the application so maybe adding those levels here could be helpful:
```
<!-- AndroidManifest.xml -->
<manifest>
    ...
    <application>
        ...
        <service...
    </application>
</manifest>
```

Reply:
do a similar thing with the meta-data section below

Reply:
Nice, added contextual tags to all 4 xml samples